src/tscore/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ add_library(
 )
 add_library(ts::tscore ALIAS tscore)
 
+include(ClangTidy)
```

Comment:
You can put this include in the top CMakeLists.txt

Reply:
that's nice!